x-pack/legacy/plugins/maps/public/layers/styles/color_gradient.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Looks like this file is a duplicate of `maps/public/layers/styles/components/color_gradient.js`.

Delete it